hw2/04-background.js

```diff
@@ -1 +1,40 @@
 // Add your code here
+const input = document.querySelector('#interval-input');
+const button = document.querySelector('#start-stop');
+button.classList.add('start');
+const bodyElement = document.querySelector('body');
+const maxValue = 255;
+let intervalID = NaN;
+bodyElement.style.backgroundColor = 'rgb(108 97 219)';
+
+function changeBackgroundColor() {
+  const red = Math.floor(Math.random() * maxValue);
+  const green = Math.floor(Math.random() * maxValue);
+  const blue = Math.floor(Math.random() * maxValue);
+  const randomColor = `rgba(${red}, ${green}, ${blue}, 0.5)`;
+
+  bodyElement.style.backgroundColor = randomColor;
+}
+
+function handleButtonClick() {
+  const interval = input.value;
+  const buttonClasses = button.classList;
+
+  if (!interval) {
+    return;
```

Comment:
You might want to alert the user that they need to put in an input if we hit this if statement.

Reply:
I like to keep them guessing